app/src/main/java/org/oppia/android/app/policies/PoliciesActivity.kt

```diff
@@ -45,4 +47,8 @@ class PoliciesActivity : InjectableAppCompatActivity() {
       }
     }
   }
+
+  override fun onRouteToPolicies(policyPage: PolicyPage) {
+    startActivity(createPoliciesActivityIntent(this, policyPage))
```

Comment:
Please add tests to verify that this activity can correctly route to each type of activity.

Reply:
Added test to verify this.